Feat/financial organizer

2jias opened the pull request:
Closes #16

DONE:


Static UI

TODO:

Add pie chart and drop down for category

![Screenshot 2024-08-01 104709](https://github.com/user-attachments/assets/3e0efc9e-5894-41f1-95de-bd88c0677394)

2jias committed:
financial-organizer changes

2jias committed:
improving UI

2jias committed:
Improvement on transaction form UI

2jias committed:
Update on transaction section UI

2jias committed:
Removing comments

2jias committed:
Update UI

- Added drop down for 'Category'
- Changed button style

Tapple214 commented:
- Inside server.js, can you add "url: financial-organizer" (please test the the url works, and if it doesnt, adjust respectively)

- inside navbar.js under components folder, please add the href to the respective icon

Tapple214 reviewed (changes requested):
Inside server.js, can you add "url: financial-organizer" (please test the the url works, and if it doesnt, adjust respectively)

inside navbar.js under components folder, please add the href to the respective icon

2jias committed:
Updates for server and navbar.js

Added "url: "/financial-organizer" inside server.js

Added respective href in navbar.js
